docs/topics/media-pipeline.rst

```diff
@@ -111,13 +111,14 @@ For the Images Pipeline, set the :setting:`IMAGES_STORE` setting::
 
    IMAGES_STORE = '/path/to/valid/dir'
 
-Supported Storage
-=================
+.. _topics-file-naming:
 
-File system storage
--------------------
+File Naming
+===========
 
-The files are stored using a `SHA1 hash`_ of their URLs for the file names.
+Default File Naming
+-------------------
+By default, Scrapy will name files using a `SHA1 hash`_ of their origination URLs.
```

Comment:
:lipstick: 

- Keep an empty line after a header.
- Maintain the parts from the original sentence that did not require a change.
- Use [“an” before SHA1](https://www.quora.com/Should-you-use-a-or-an-before-an-acronym-that-begins-with-the-letter-S) while at it. Ideally it should also be SHA-1, but that requires changing other lines as well, so up to you.

```suggestion
Default File Naming
-------------------

By default, files are stored using an `SHA1 hash`_ of their URLs for the file names.
```